style.css

```diff
@@ -91,5 +101,162 @@ main .social svg {
 }
 
 main .action {
-  margin-bottom: 58px;
+  margin-bottom: 22px;
+}
+
+.section-heading {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+
+.section-heading h1 {
+  margin: 0 20px;
+  color: #172b4d;
+}
+
+.section-heading .indicator {
+  width: 48px;
+  height: 4px;
+  border-radius: 24px;
+  background-color: #36b37e;
+}
+
+.recent-works .card-container {
+  display: grid;
+  justify-items: center;
+  color: #3a4a42;
+  text-align: center;
+}
+
+.recent-works .image {
+  background-color: #ebf0ee;
+  height: 250px;
+  width: 100%;
+}
+
+.card {
+  background-color: whitesmoke;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 26px;
+  border-radius: 8px;
+  overflow: hidden;
+  border: 1px solid #d0d9d4;
+  width: 327px;
+}
+
+.card h2 {
+  font-weight: 600;
+}
+
+.card .inner-box {
+  width: 100%;
+  border-radius: 8px;
+  background-color: #fff;
+  height: 222px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.card ul {
+  list-style-type: none;
+  margin: 0 11px;
+  display: flex;
+  gap: 8px;
+}
+
+.card ul li {
+  background-color: #ebf0ee;
+  border-radius: 4px;
+  padding: 8px 12px;
+  font-weight: 600;
+  font-size: 12px;
+  width: max-content;
+}
+
+button {
+  background-color: #36b37f;
+  border: none;
+  padding: 12px;
+  color: white;
+  font-weight: 500;
+  font-size: 17px;
+  font-family: Inter, 'Times New Roman', Times, serif;
+  border-radius: 4px;
+  text-align: justify;
+}
+
+button:active {
+  background-color: #008552;
+}
+
+button:hover {
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.about .description {
+  margin: 0 12px;
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
+
+.about .card-container {
+  margin-top: 59px;
+}
+
+.about .card {
+  background-color: rgba(235, 240, 238, 1);
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] CSS code best practices the DRY concept. It's always good to avoid repeating the same lines of code. Here you can wrap these two lines in a class that you can use whenever it's needed in the HTML code instead of repeating the same lines many times in the css. 
 ` .flex {
  display: flex;
  flex-direction: column;`
}
You can do the same for any repeated code. It will make your code better.